numpy/random/_generator.pyx

```diff
@@ -396,6 +396,43 @@ cdef class Generator:
         out : ndarray or scalar
             Drawn samples from the parameterized beta distribution.
 
+        Examples
+        -------- 
+        The beta distribution has mean a/(a+b). If ``a == b`` and both 
+        are > 1, the distribution is symmetric with mean 0.5.
+
+        >>> rng = np.random.default_rng()
+        >>> a, b, size = 2.0, 2.0, 10000
+        >>> sample = rng.beta(a=a, b=b, size=size)
+        >>> np.mean(sample)
+        0.5047328775385895  # may vary
+        
+        Otherwise the distribution is skewed left or right according to
+        whether ``a`` or ``b`` is greater. The distribution is mirror
+        symmetric. See for example:
+        
+        >>> a, b, size = 2, 7, 10000
+        >>> sample_left = rng.beta(a=a, b=b, size=size)
+        >>> sample_right = rng.beta(a=b, b=a, size=size)
+        >>> m_left, m_right = np.mean(sample_left), np.mean(sample_right)
+        >>> print(m_left, m_right)
+        0.2238596793678923 0.7774613834041182  # may vary
+        >>> print(m_left - a/(a+b))
+        0.001637457145670096  # may vary
+        >>> print(m_right - b/(a+b))
+        -0.0003163943736596009  # may vary
+
+        Display the histogram of the two samples:
+        >>> import matplotlib.pyplot as plt
```

Comment:
I think you need a blank line , see the [resulting documentation](https://output.circle-artifacts.com/output/job/eed957a2-ec55-4894-bf37-20eafda82247/artifacts/0/doc/build/html/reference/random/generated/numpy.random.Generator.beta.html#numpy.random.Generator.beta)

Reply:
Thanks.I should have spotted that. I added an empty line